CorsixTH/CorsixTH.lua

```diff
@@ -17,6 +17,14 @@ for _, arg in ipairs({...}) do
   end
 end
 
+-- On Mac OS X add the local luarocks folder to package.cpath.
+if (io.popen"uname":read'*l' == "Darwin") then
```

Comment:
Should use brackets for popen and read. Could potentially be used everywhere not just OSX, so maybe this outer condition isn't necessary.